packages/web-shared/components/ContentSingle.js

```diff
@@ -11,14 +11,77 @@ import FeatureFeedComponentMap from './FeatureFeed/FeatureFeedComponentMap';
 import { add as addBreadcrumb, useBreadcrumbDispatch } from '../providers/BreadcrumbProvider';
 import { set as setModal, useModal } from '../providers/ModalProvider';
 
-import { Box, Loader, Longform, H3, ContentCard, BodyText, ShareButton } from '../ui-kit';
+import {
+  Box,
+  Loader,
+  Longform,
+  H3,
+  ContentCard,
+  BodyText,
+  ShareButton,
+  Icons,
+  PhospherIcon,
+} from '../ui-kit';
 import { useFeatureFeed, useHTMLContent, useVideoMediaProgress } from '../hooks';
 import { Title, ParentTitle, ParentSummary } from './ContentSingle.styles';
 
 import VideoPlayer from './VideoPlayer';
 
 import InteractWhenLoaded from './InteractWhenLoaded';
 import TrackEventWhenLoaded from './TrackEventWhenLoaded';
+import styled from 'styled-components';
+
+const infoDivider = (
+  <BodyText color="text.tertiary" mx="xs" display={{ xs: 'none', sm: 'block' }}>
+    |
+  </BodyText>
+);
+
+const CalendarIconWrapper = styled.span`
+  margin-right: ${({ theme }) => theme.space.xxs};
+  display: inline-block;
+  @media (min-width: ${({ theme }) => theme.breakpoints.sm}) {
+    display: none;
+  }
+`;
+
+const CalendarIcon = ({ name, size }) => (
+  <CalendarIconWrapper>
+    <PhospherIcon name={name} size={size} />
+  </CalendarIconWrapper>
+);
+
+const CalendarDataContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+function CalendarData({ start, end, location }) {
+  return (
+    <>
+      {location ? (
+        <BodyText color="text.secondary" mb={'xxs'}>
+          <CalendarIcon name="globe" size={16} />
```

Comment:
I think either ditch the size here, or include it on the other two implementations.